### PR TITLE
Implement sidecar containers

### PR DESCRIPTION
**Pull request by adibrastegarnia**
This PR allows subsystems to add sidecar containers to the subsystems. The first use is to deploy onos-config model plugins as sidecar containers. To test, run:
`
onit create cluster `

and the expected output is:

```
kubectl get pods -n onos  -w
NAME                                 READY   STATUS    RESTARTS   AGE
atomix-controller-7fc97fbb88-z9n2n   1/1     Running   0          2m7s
database-1-0                         1/1     Running   0          2m5s
onos-config-764984d7b6-qnbbv         5/5     Running   0          2m4s
onos-ran-674d47bfd4-hdf2d            1/1     Running   0          2m4s
onos-topo-75696c7b6-7hjrq            1/1     Running   0          2m4s
```

### CODE REVIEW

File: pkg/onit/cluster/container.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	c.env = env
+}
+
+// AddEnv adds an environment variable
+func (c *Container) AddEnv(name, value string) {
+	c.env[name] = value
+}
+
+// VolumeMount volume mount data structure
+type VolumeMount struct {
```
Comment (kuujo) [inline pkg/onit/cluster/container.go]: Can we use the k8s `VolumeMount` here?

File: pkg/onit/cluster/container.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	name string
+	path string
+}
+
+// Container provides methods for adding sidecar containers to a deployment
+type Container struct {
```
Comment (kuujo) [inline pkg/onit/cluster/container.go]: It feels like we should name this abstraction `Sidecar` to distinguish it from the pod architecture which takes a generic list of containers. In this case, we're saying every pod has a service container and a set of optional sidecars

File: pkg/onit/cluster/container.go (hunk 1 above)
Comment (adibrastegarnia) [reply pkg/onit/cluster/container.go]: Done

File: pkg/onit/cluster/container.go (hunk 2 above)
Comment (adibrastegarnia) [reply pkg/onit/cluster/container.go]: Done.

**Pull request merged**